### PR TITLE
added router listening for workouts-list to the server file that runs…

### PR DESCRIPTION
**Pull request by jakedoublev**
changed filename of router middleware file from DBConnection to queriesRoutedToDB
added middleware to query the DB for all entries from the workout_card table
sends workout card data as an object with a key of workoutsList and a key value of an array of the queried workout objects back to the front-end as JSON with the status 200
error handles for an error when querying and for no data returned from the query (empty obj)

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -5,22 +5,22 @@ const path = require("path");
...
  * handle parsing request body
  */
 app.use(express.json());
 
-//handle page not found
-app.use((req, res) =>
-  res.status(404).send("This is not the page you're looking for...")
-);
+//handle workouts-list query for workout cards data
+app.get("/workouts-list", queriesRouter.getWorkoutsList, (req, res) => {
```
Comment (sktatsuno) [inline server/server.js:18]: workouts-list route does not match workoutList proxy

File: webpack.config.js [hunk 2]
```diff
@@ -44,5 +44,9 @@ module.exports = {
       //directory: path.resolve(__dirname, 'build'),
       publicPath: "/build",
     },
+    proxy: {
+      // "*": "https://[::1]:3000",
+      "/workoutsList/**": "http://localhost:3000",
```
Comment (sktatsuno) [inline webpack.config.js]: workouts-list route does not match workoutList proxy

**Review by sktatsuno — CHANGES_REQUESTED** [review]
workouts-list route name does not match workoutsList proxy. Left inline comments hopefully you can see those

File: webpack.config.js (hunk 2 above)
Comment (jakedoublev) [reply webpack.config.js]: fixed

File: server/server.js (hunk 1 above)
Comment (jakedoublev) [reply server/server.js:18]: fixed

**Review by jakedoublev — COMMENTED** [review]
comments resolved. files changed.

**Review by sktatsuno — APPROVED** [review]
Looks good now

**Pull request merged**